.readthedocs.yaml

```diff
@@ -14,6 +14,9 @@ build:
       - git fetch --unshallow || true
     pre_install:
       - git update-index --assume-unchanged doc/conf.py ci/requirements/doc.yml
+      - export NO_COLOR=1
+      - export TERM=dumb
+      - export FORCE_COLOR=0
```

Comment:
not sure why is this needed?

Reply:
Disables color codes on RTD builds so that errors are readable. I pushed this change